### PR TITLE
Validation error message

### PR DESCRIPTION
**Pull request by ghost**
# Description
Fixes bug in input validation decorator which causes error message to flicker.

# Before
validation_error_message_flicker

# After
validation_error_message_flicker_after_fix

### CODE REVIEW

File: src/utils/decorators/input-validation/input-validation.js [hunk 1]
```diff
@@ -633,7 +634,10 @@ const InputValidation = (ComposedComponent) => {
 
       const errorMessage = (this.state.messageLocked || this.state.messageShown) && (
         <Portal key='1' onReposition={ this.positionMessage }>
-          <div className='common-input__message-wrapper'>
+          <div
+            className='common-input__message-wrapper'
+            style={ { pointerEvents: 'none' } }
```
Comment (ianoxley) [inline src/utils/decorators/input-validation/input-validation.js]: It'd be better to add `pointer-event` to the `input.scss` file where the `.common-input__message-wrapper` is class is defined.

File: webpack.config.js [hunk 2]
```diff
@@ -5,5 +5,6 @@ module.exports = require('carbon-factory/webpack.config')({
   port: 8095,
   serverBase: './deploy',
   parcelifyPaths: [path.resolve(process.cwd(), './demo')],
-  singlePageApp: true
+  singlePageApp: true,
+  devtool: 'eval-source-map'
```
Comment (ianoxley) [inline webpack.config.js]: Were these changes included by mistake?

File: src/utils/decorators/input-validation/input-validation.js [hunk 3]
```diff
@@ -712,6 +716,7 @@ const InputValidation = (ComposedComponent) => {
       return fieldProps;
     }
 
+
```
Comment (ianoxley) [inline src/utils/decorators/input-validation/input-validation.js]: Remove the extra blank line.

**Comment by ianoxley** [comment]
It'd be good to include an `after` screenshot above too to compare the behaviour with and without your fix.

File: src/utils/decorators/input/input.scss [hunk 4]
```diff
@@ -117,6 +117,7 @@
   top: 0;
   position: absolute;
   width: 260px;
+  pointer-events: none;
```
Comment (ianoxley) [inline src/utils/decorators/input/input.scss]: Can you sort the CSS properties alphabetically please?

**Comment by andrewjtait** [comment]
Tested in demo site, flicker no longer occurs and message is still interactive ✅

**Pull request merged**